GVFS/GVFS.UnitTests/Common/Database/GVFSDatabaseTests.cs

```diff
@@ -87,11 +87,25 @@ private void TestGVFSDatabase(Action<GVFSDatabase> testCode, bool throwException
 
             mockCommand2.Setup(x => x.Dispose());
 
+            Mock<IDbCommand> mockCommand3 = new Mock<IDbCommand>(MockBehavior.Strict);
+            mockCommand3.SetupSet(x => x.CommandText = "CREATE TABLE IF NOT EXISTS [IncludedFolder] (path TEXT PRIMARY KEY COLLATE NOCASE) WITHOUT ROWID;");
+            if (throwException)
+            {
+                mockCommand3.Setup(x => x.ExecuteNonQuery()).Throws(new Exception("Error"));
+            }
+            else
+            {
+                mockCommand3.Setup(x => x.ExecuteNonQuery()).Returns(1);
+            }
+
+            mockCommand3.Setup(x => x.Dispose());
+
             List<Mock<IDbConnection>> mockConnections = new List<Mock<IDbConnection>>();
             Mock<IDbConnection> mockConnection = new Mock<IDbConnection>(MockBehavior.Strict);
             mockConnection.SetupSequence(x => x.CreateCommand())
                           .Returns(mockCommand.Object)
-                          .Returns(mockCommand2.Object);
+                          .Returns(mockCommand2.Object)
+                          .Returns(mockCommand3.Object);

```

Comment:
Should there be any validation for `mockCommand3` like we have for the other mock commands below?

```
            mockCommand.Verify(x => x.Dispose(), Times.Once);
            mockCommand2.Verify(x => x.Dispose(), Times.Once);
            mockConnections.ForEach(connection => connection.Verify(x => x.Dispose(), Times.Once));	            
            
            mockCommand.VerifyAll();
            mockCommand2.VerifyAll();
```

(Full disclosure, I am not exactly sure how these `mockCommand`s work in this test)